### ISSUE TITLE
Click on a skin color -> the emojis change color accordingly

### ISSUE THREAD

**Issue by amivaleo** [Maestroschan/emoji-selector-for-gnome#44]
Hi!

Consider the category "Peoples & Clothing". There you gave us the chance to select the skin color (and the gender) of several emojis.
Is it possible to have the following behaviour:
click on a skin color -> the emojis in the list changes color accordingly
?

**Comment by Maestroschan** [comment]
Yes, it was the idea i had in mind when i began the skin color support, but i'll do it later because it requires a lot of changes
